### PR TITLE
Removes error message while shunting [fix]

### PR DESCRIPTION
**Pull request by 4dplanner**

### CODE REVIEW

**Comment by 4dplanner** [comment]
Will fix up when I get back

File: code/modules/mob/living/silicon/ai/freelook/eye.dm [hunk 1]
```diff
@@ -160,11 +160,13 @@
...
-		to_chat(src, "ERROR: Eyeobj not found. Creating new eye...")
-		create_eye()
+	if(isturf(loc)) //not shunted
+		if(!eyeobj || !eyeobj.loc || QDELETED(eyeobj))
+			to_chat(src, "ERROR: Eyeobj not found. Creating new eye...")
+			create_eye()
 
-	eyeobj.setLoc(loc)
+	if(eyeobj)
+		eyeobj.setLoc(loc)
```
Comment (ninjanomnom) [inline code/modules/mob/living/silicon/ai/freelook/eye.dm]: `eyeobj?.setLoc(loc)`

File: code/modules/mob/living/silicon/ai/freelook/eye.dm [hunk 2]
```diff
@@ -160,11 +160,13 @@
 	cameraFollow = null
 	unset_machine()
 
-	if(!eyeobj || !eyeobj.loc || QDELETED(eyeobj))
-		to_chat(src, "ERROR: Eyeobj not found. Creating new eye...")
-		create_eye()
+	if(isturf(loc)) //not shunted
+		if(!eyeobj || !eyeobj.loc || QDELETED(eyeobj))
```
Comment (ninjanomnom) [inline code/modules/mob/living/silicon/ai/freelook/eye.dm]: `if(isturf(loc) && (QDELETED(eyeobj) || !eyeobj.loc))`

File: code/modules/mob/living/silicon/ai/life.dm [hunk 3]
```diff
@@ -19,8 +19,9 @@
 			// messenging the client
 			malfhacked(malfhack)
 
-		if(!eyeobj || QDELETED(eyeobj) || !eyeobj.loc)
-			view_core()
+		if(isturf(loc)) //not shunted
+			if(!eyeobj || QDELETED(eyeobj) || !eyeobj.loc)
```
Comment (ninjanomnom) [inline code/modules/mob/living/silicon/ai/life.dm]: `if(isturf(loc) && (QDELETED(eyeobj) || !eyeobj.loc))`

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Pull request merged**